giraffe/src/components/geo/GeoMarker.tsx

```diff
@@ -0,0 +1,13 @@
+import {withLeaflet} from 'react-leaflet'
+import {Component} from 'react'
+
+class GeoMarker extends Component {
+  constructor(props) {
```

Comment:
Same here. Function Component with Hooks, please.

Reply:
It is done.